HealthcareSystem/Backend/Model/Manager/TypeOfEquipment.cs

```diff
@@ -1,14 +1,24 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+/***********************************************************************
+ * Module:  TypeOfConsumable.cs
+ * Author:  Jelena Budisa
+ * Purpose: Definition of the Class TypeOfConsumable
+ ***********************************************************************/
+
+using System;
 
 namespace Model.Manager
 {
     public enum TypeOfEquipment
     {
-        CONSUMABLE = 0,
-        NON_CONSUMABLE = 1
+        NEEDLE = 0,
+        MASK = 1,
+        BEND = 2,
+        DESINFECTANT = 3,
+        BED = 4,
+        TABLE = 5,
+        CHAIR = 6,
+        COMPUTER = 7,
+        INSTRUMENT = 8
```

Comment:
Are you sure this would be the right approach for this problem since when writing the name of an equipment object you could easily make a typo and instead of bend get a bed? I cannot find the Specialty class, is it merged into develop?

Reply:
Yeah, I think Specialty is on develop, merged just yesterday. And I'm not exactly sure when the issue with typos could happen - do you mean when selecting an equipment type? And the answer to that is that with decent HCI it would never be typed in those cases, you'd still have the user select one of the existing equipment types from a combo box or something. The only difference is that instead of having a fixed enumeration tell you what those existing types are, you would query the database to find out.
And if you mean that typos could happen while administrating the equipment types and adding new ones (which would be done either by the database admin, or exposed to a user with admin privileges like the manager for example) - yes, of course typos could happen, but I fail to see an issue with that. Whoever is administrating would be able to see the typo and just fix it whenever. 
But yeah, generally speaking, it is 100% the right approach pretty much whenever you have a type-instance sort of relationship. Enumerations are fine when you have something with values that won't change unless really extreme circumstances happen, and that also doesn't have that many values, like state of an object or something. But for something like this, where it's absolutely reasonable that types can be added and removed at will, an enumeration just doesn't cut it. 
Of course, you don't have to implement this if you don't have the time or don't like it as a design choice (although it's objectively a better design choice than what we currently have).

Reply:
I wanted to recommit yesterday, but Milena has already done it.
I believe enums could make some use cases smoother, but of course, this approach is always available and could be really useful for a wide range of cases. Since Milena has already made changes, I believe we could resolve this part of the conversation.